addons/portal/static/src/chatter/core/composer_patch.js

```diff
@@ -0,0 +1,12 @@
+import { Composer } from "@mail/core/common/composer";
+import { _t } from "@web/core/l10n/translation";
+import { patch } from "@web/core/utils/patch";
+
+patch(Composer.prototype, {
+    get placeholder() {
+        if (this.thread && this.thread.model !== "discuss.channel") {
+            return _t("Write a message…");
+        }
```

Comment:
Again, this doesn't look correct. We can't change the behavior of base mail code in `core` when portal is installed, it makes no sense.
And even in portal specific bundle I don't see the value.

Reply:
The portal chatter bundle is included in the frontend bundle and is completely separate from the backend bundle. So any change to the core or whatever in this bundle won't affect the backend chatter (a lot of tests would fail otherwise). It could only affect livechat, which I tried to prevent by filtering threads by `discuss.channel`. 
And without this change, the default placeholder in chatter composer would be `Log an internal note...`.

Reply:
`core` should always be included in all bundles, that is our guideline with these folders. Please change the folder name to keep the structure correct.

If you need a portal only, it should be in the folder appropriately named, instead of core.